vividus-to-zephyr-for-jira-exporter/src/main/java/org/vividus/facade/zephyr/ZephyrFacade.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.facade.zephyr;
+
+import static org.apache.commons.lang3.Validate.notBlank;
+import static org.apache.commons.lang3.Validate.notEmpty;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.vividus.facade.jira.JiraConfiguration;
+import org.vividus.facade.jira.client.IJiraClient;
+import org.vividus.util.json.JsonPathUtils;
+
+public class ZephyrFacade implements IZephyrFacade
```

Comment:
instead of using json path in many placeless can't we just use pojos deserialized from jsons we get?